src/features/modals/CreateToken/CreateTokenBody/TokenSummary/TokenSummary.module.scss

```diff
@@ -0,0 +1,32 @@
+.TokenSummaryRow {
+	display: flex;
+	margin-top: 10px;
+
+	.TokenSummaryColumn {
+		display: flex;
+		flex-direction: column;
+
+		.TokenSummaryHeading {
+			margin-top: 0;
+		}
+
+		.TokenSummaryMediaInput {
+			height: 200px;
+			width: 200px;
+			margin-right: 24px;
+			border-radius: 8px !important;
```

Comment:
`rem` values here. Have mentioned a couple times in this review - will stop commenting on it now but if you could change `px` values to `rem` that would be beaut.